webcrypto/goja.go

```diff
@@ -29,6 +33,34 @@ func exportArrayBuffer(rt *goja.Runtime, v goja.Value) ([]byte, error) {
 	return ab.Bytes(), nil
 }
 
+// traverseObject traverses the given object using the given fields and returns the value
+// at the end of the traversal. It assumes that all the traversed fields are Objects.
+func traverseObject(rt *goja.Runtime, src *goja.Object, fields ...string) (goja.Value, error) {
+	var value goja.Value = src
```

Comment:
```suggestion
func traverseObject(rt *goja.Runtime, value goja.Value, fields ...string) (goja.Value, error) {
```
I was going to recommend this at the call site, but there is no need for you to make the value into an object if all of that can be done here to begin with.